### PR TITLE
feat: add parameters in helm to disable/enable nfd-master and nfd-worker

### LINKED ISSUE #1395: Helm: add parameters for disabling nfd-master and nfd-worker
**What would you like to be added**:

Add `master.enable` and `worker.enable` parameters that allow users to skip deployment of nfd-master and nfd-worker.

**Why is this needed**:

Enable use cases like
- only run nfd-master: only custom labels from 3rd party extensions (via NodeFeature CRD API) are needed
- only run nfd-topology-updater

### PR DESCRIPTION
**Pull request by AhmedGrati**
Resolves #1395.

### CODE REVIEW

File: deployment/helm/node-feature-discovery/values.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ namespaceOverride: ""
 enableNodeFeatureApi: true
 
 master:
+  enabled: true
```
Comment (marquiz) [inline deployment/helm/node-feature-discovery/values.yaml:16]: We have `gc.enable`, `topologyUpdater.enable`, `prometheus.enable` so for consistency it'd be good to have this `master.enable`

```suggestion
  enable: true
```

File: deployment/helm/node-feature-discovery/values.yaml [hunk 2]
```diff
@@ -132,6 +133,7 @@ master:
                 values: [""]
 
 worker:
+  enabled: true
```
Comment (marquiz) [inline deployment/helm/node-feature-discovery/values.yaml:136]:
```suggestion
  enable: true
```

**Review by marquiz — CHANGES_REQUESTED** [review]
Thankx @AhmedGrati for this enhancement. One comment regarding the parameter naming below.

In addition, make creation of the RBAC objects (clusterrole, clusterrolebinding, role, rolebinding and serviceaccount) conditionally dependent on the new parameters.

**Review by marquiz — APPROVED** [review]
Good improvement @AhmedGrati 
/assign @ArangoGutierrez

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Pull request merged**